src/libs/_tests/runtime-utils.test.js

```diff
@@ -1,9 +1,28 @@
 import { tools } from 'src/components/notebook-utils'
 import {
-  getAnalysesDisplayList, getCurrentApp, getCurrentAppIncludingDeleting, getCurrentPersistentDisk, getDiskAppType, workspaceHasMultipleApps, workspaceHasMultipleDisks
+  getAnalysesDisplayList, getCostDisplayForDisk, getCostDisplayForTool, getCurrentApp, getCurrentAppDataDisk, getCurrentAppIncludingDeleting, getDiskAppType, workspaceHasMultipleApps, workspaceHasMultipleDisks
 } from 'src/libs/runtime-utils'
 
 
+jest.mock('src/data/machines', () => {
+  const originalModule = jest.requireActual('src/data/machines')
+
+  return {
+    ...originalModule,
+    regionToPrices: [
+      {
+        name: 'US-CENTRAL1', monthlyStandardDiskPrice: 0.04, monthlySSDDiskPrice: 0.17, monthlyBalancedDiskPrice: 0.1,
+        n1HourlyGBRamPrice: 0.004237, n1HourlyCpuPrice: 0.031611, preemptibleN1HourlyGBRamPrice: 0.000892, preemptibleN1HourlyCpuPrice: 0.006655,
+        t4HourlyPrice: 0.35, p4HourlyPrice: 0.6, k80HourlyPrice: 0.45, v100HourlyPrice: 2.48, p100HourlyPrice: 1.46,
+        preemptibleT4HourlyPrice: 0.11, preemptibleP4HourlyPrice: 0.216, preemptibleK80HourlyPrice: 0.0375,
+        preemptibleV100HourlyPrice: 0.74, preemptibleP100HourlyPrice: 0.43
+      }
+    ]
+  }
+})
+
+jest.clearAllMocks(true)
```

Comment:
I am curious why this is needed.... could it be that any weirdness you were seeing could be fixed by changing regionToPrices export to an arrow function that returns that array of info instead of the array directly?